launcher-scripts/Linux-launcher

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+
+if [ ! -d "jdk8u212-b03" ] 
+then
+    tar -xzf jre.tar.gz
+fi
+currentPath=`pwd`
+echo "export JAVA_HOME=$currentPath/jdk8u212-b03/" >>~/.bashrc
+echo "export PATH=$JAVA_HOME/bin:$PATH" >>~/.bashrc
```

Comment:
No no no... We're not installing a new JRE for them, we just want to use the one in the current directory.

So this would just be:

    export JAVA_HOME=`pwd`/jdk8u212-bo3/

And no need to modify path.